python/tank/descriptor/io_descriptor/shotgun_entity.py

```diff
@@ -66,7 +66,6 @@ def __init__(self, descriptor_dict, sg_connection):
             optional=["project_id"]
         )
 
-        self._sg_connection = sg_connection
```

Comment:
[minor] Can you make the same notation here in the init docs as you did in the appstore descriptor class? It's nice to know that the provided sg handle won't be used if it is passed in.